doc/application_workflow/harbor_registry/harbor_with_static_ip.md

```diff
@@ -0,0 +1,209 @@
+Common Name (e.g. server FQDN or YOUR name) []:<Harbor static ip HERE!>
+Email Address []:
+
+Please enter the following 'extra' attributes
+to be sent with your certificate request
+A challenge password []:password
+An optional company name []:
+loc@Devbox:~/static_certs$ ls
+ca.crt  ca.key  server.csr  server.key
+```
+**Note above, we specify the static IP that we assigned to Harbor in the common name field.**  We must do this for standard docker and VIC Engine to recognize the certificate's signing CA that we generated above.
+
+Sign the certificate with the CA cert:
+
+```
+loc@Devbox:~/static_certs$ mkdir demoCA
+loc@Devbox:~/static_certs$ cd demoCA/
+loc@Devbox:~/static_certs/demoCA$ touch index.txt
+loc@Devbox:~/static_certs/demoCA$ echo '01' > serial
+loc@Devbox:~/static_certs/demoCA$ cd ..
+loc@Devbox:~/static_certs$ ls
+ca.crt  ca.key  demoCA  server.csr  server.key
+
+loc@Devbox:~/static_certs$ echo subjectAltName = IP:<Harbor static ip HERE!> > extfile.cnf
+
+loc@Devbox:~/static_certs$ openssl ca -in server.csr -out server.crt -cert ca.crt -keyfile ca.key -extfile extfile.cnf -outdir .
+Using configuration from /usr/lib/ssl/openssl.cnf
+Check that the request matches the signature
+Signature ok
+Certificate Details:
+        Serial Number: 1 (0x1)
+        Validity
+            Not Before: Nov 18 00:12:14 2016 GMT
+            Not After : Nov 18 00:12:14 2017 GMT
+        Subject:
+            countryName               = US
+            stateOrProvinceName       = California
+            organizationName          = VMware
+            organizationalUnitName    = vic
+            commonName                = <Harbor static ip HERE!>
+        X509v3 extensions:
+            X509v3 Subject Alternative Name: 
+                IP Address:<Harbor static ip HERE!>
+Certificate is to be certified until Nov 18 00:12:14 2017 GMT (365 days)
+Sign the certificate? [y/n]:y
+
+
+1 out of 1 certificate requests certified, commit? [y/n]y
+Write out database with 1 new entries
+Data Base Updated
+
+loc@Devbox:~/static_certs$ ls
+01.pem  ca.crt  ca.key  demoCA  extfile.cnf  server.crt  server.csr  server.key
+
+loc@Devbox:~/static_certs$ rm *.csr
+```
+<br><br>
+
+## Install Harbor on a vCenter installation
+
+The first part of this install and setup can have many variations.  The important part is that we must have an FQDN for the Harbor server and we must know the assigned IP addressed for that server.  Future version of Harbor will not require both.  Future version will also allow the server to be installed with either FQDN or static IP, but for now, it must be installed with an FQDN.
+
+Depending on your vCenter setup, your steps here maybe different.  For our setup, we will first deploy Harbor with the OVA and fill out ONLY vApp parameters for the passwords.  Once the VM is up and running, our DNS server will assign an FQDN and IP address to the Harbor VM.  We will use the vSphere web client in this example.
```

Comment:
s/maybe/may be

Also may want to indicate that your datastore must be at least >= 60GB in order to use the OVA (I ran into this issue myself).